This adds a library necessary for sparc. 
This will not be sent upstream because it would be difficult to
add a sparc-only ifdef condition for this.

diff --git a/browser/installer/package-manifest.in b/browser/installer/package-manifest.in
--- a/browser/installer/package-manifest.in
+++ b/browser/installer/package-manifest.in
@@ -804,6 +804,7 @@
 bin/libfreebl_32fpu_3.so
 bin/libfreebl_32int_3.so
 bin/libfreebl_32int64_3.so
+bin/cpu/sparcv8plus/libnspr_flt4.so
 #endif
 
 ; [Updater]
